scripts/openmic_split.py

```diff
@@ -34,8 +35,11 @@ def process_args(args):
                         help='Path to sparse-labels.csv',
                         type=str)
 
+    parser.add_argument('--dupes', dest='dupe_file', type=str,
+                        help='Path to track de-duplication index')
+
     parser.add_argument('-s', '--seed', dest='seed',
-                        default=20180821,
+                        default=20180903,
```

Comment:
nitwarning: if we're going to use dates, maybe we can pull this from date-time so we won't have to keep bumping it? otherwise it's probably fine to leave it as some pseudo-arbitrary number, e.g. 12345?

Reply:
I was taking the date of writing as a pseudo-arbitrary number for the seed, but one that's unlikely to be randomly reproduced elsewhere.  It's a silly convention, and I for sure didn't need to change the seed here, but there's no harm in leaving it here.

Current date-time is exactly what I don't want, since that won't be fixed on subsequent runs.